shot-android/src/main/java/com/karumi/shot/FragmentScenarioUtils.kt

```diff
@@ -0,0 +1,27 @@
+package com.karumi.shot
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.testing.FragmentScenario
+import com.karumi.shot.ActivityScenarioUtils.waitForActivity
+
+/**
+ * Shot needs to be executed from the instrumentation thread to be able to extract
+ * all the test metadata needed to record and verify screenshots. That's why we've created this
+ * extension to be able to get the fragment instance from the instrumentation thread instead
+ * of running Shot from the app target thread. I hope we can find a better solution in the future.
```

Comment:
I copy-pasted this comment from [ActivityScenarioUtils](https://github.com/Karumi/Shot/blob/260f8d5c38f815f6d3f0ef64feb0234aa4375973/shot-android/src/main/java/com/karumi/shot/ActivityScenarioUtils.kt).

We move both extension to the same file and keep an explanation on top of it.